rpcserver.go

```diff
@@ -3296,6 +3296,10 @@ func (r *rpcServer) PendingChannels(ctx context.Context,
 			}
 			channel.NumForwardingPackages = int64(len(fwdPkgs))
 
+			channel.RemoteBalance = int64(
+				historical.LocalCommitment.RemoteBalance.ToSatoshis(),
```

Comment:
Balance should be correct as this would be an unconfirmed force close, and only the force-closer would have awareness of an unconfirmed force close. We don't persist pending commitment updates as the recipient so `LocalCommitment` is correct here

Reply:
I tested this with both incoming and outgoing HTLCs and the balance seems to be as expected.